packages/aws/data_stream/awshealth/fields/fields.yml

```
@@ -0,0 +1,116 @@
+- name: aws
+  type: group
+  fields:
+    - name: awshealth
+      type: group
+      description: >
+        AWS Health metrics
+
+      fields:
+        - name: affected_entities_others
+          type: float
+          metric_type: gauge
+          description: >
+            The number of affected resources related to the event whose status cannot be verified.
+
+        - name: affected_entities_pending
+          type: float
+          metric_type: gauge
+          description: >
+            The number of affected resources that may require action.
+
+        - name: affected_entities_resolved
+          type: float
+          metric_type: gauge
+          description: >
+            The number of affected resources that do not require any action.
+
+        - name: end_time
+          type: date
+          description: >
+            The date and time when the event ended. Some events may not have an end date.
+
+        - name: event_arn
+          type: keyword
+          dimension: true
+          description: >
+            The unique identifier for the event. The event ARN has the format arn:aws:health:event-region::event/SERVICE/EVENT_TYPE_CODE/EVENT_TYPE_PLUS_ID.
+
+        - name: event_scope_code
+          type: keyword
+          description: "This parameter specifies whether the Health event is a public Amazon Web Service event or an account-specific event. Allowed values are PUBLIC, ACCOUNT_SPECIFIC, or NONE. \n"
```

```suggestion
          description: This parameter specifies whether the Health event is a public Amazon Web Service event or an account-specific event. Allowed values are PUBLIC, ACCOUNT_SPECIFIC, or NONE.
```
Also please keep the space between the fields consistent.

---

Addressed.